<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>GroupsDialogUI</class>
 <widget class="QDialog" name="GroupsDialogUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit User Groups</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>4</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="RecordTreeView" name="mGroupTree" />
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="mNewGroup" >
         <property name="text" >
          <string>&amp;New Group</string>
         </property>
         <property name="shortcut" >
          <string>Alt+N</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mDeleteGroup" >
         <property name="text" >
          <string>&amp;Delete Group</string>
         </property>
         <property name="shortcut" >
          <string>Alt+D</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType" >
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>81</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mOK" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
       <property name="shortcut" >
        <string>Alt+O</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>RecordTreeView</class>
   <extends>QTreeView</extends>
   <header>recordtreeview.h</header>
   <container>0</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>mOK</sender>
   <signal>clicked()</signal>
   <receiver>GroupsDialogUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>247</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel" >
     <x>169</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mCancel</sender>
   <signal>clicked()</signal>
   <receiver>GroupsDialogUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>374</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel" >
     <x>400</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
